src/ffi_ptr_ext.rs

```diff
@@ -0,0 +1,28 @@
+use crate::{ffi, instance::Py2, PyAny, PyResult, Python};
+
+mod ffi_ptr_ext_sealed {
```

Comment:
Here and below, it doesn't seem necessary to repeat the parent module name, i.e. `mod sealed {` should work just as well.